pootle/static/js/browser.js

```diff
@@ -175,37 +175,50 @@ function fixDropdowns() {
 }
 
 
+function formatResult(text, term) {
+  const match = text.toUpperCase().indexOf(term.toUpperCase());
+  const tl = term.length;
+  const markup = [];
+
+  if (match < 0) {
+    markup.push(_.escape(text));
```

Comment:
What about just `return _.escape(text);` ? That way the whole function can be simplified like:

```javascript
function formatResult(text, term) {
  const match = text.toUpperCase().indexOf(term.toUpperCase());

  if (match < 0) {
    return _.escape(text);
  }

  const tl = term.length;
  const markup = [];

  markup.push(_.escape(text.substring(0, match)));
  markup.push('<span class="select2-match">');
  markup.push(_.escape(text.substring(match, match + tl)));
  markup.push('</span>');
  markup.push(_.escape(text.substring(match + tl, text.length)));

  return markup.join('');
}
```